dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/validation/EventImportValidationTest.java

```diff
@@ -89,9 +90,14 @@ void setUp() throws IOException {
             params, fromJson("tracker/validations/enrollments_te_enrollments-data.json")));
   }
 
+  @AfterEach
```

Comment:
Just curious. Why do we rely on AfterEach instead of BeforeEach?

Reply:
Because I forgot to change it.